projects/plugins/jetpack/class.jetpack.php

```diff
@@ -694,14 +694,14 @@ private function __construct() {
 		 */
 		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-gutenberg.php';
 		add_action( 'plugins_loaded', array( 'Jetpack_Gutenberg', 'load_independent_blocks' ) );
```

Comment:
Also, without moving this too you won't really fix https://github.com/Automattic/jetpack/issues/39351. Both of these call the `should_load()` method that checks the `jetpack_blocks_disabled` option.

And moving this will likely affect many more blocks. I tried Business Hours and Tiled Gallery, and both broke when moving this line with the other.

Reply:
Thanks. I think this as a bandaid for moving the module is just worse than moving it. Looking at 39449 as an alternative approach.